components/create_comment/index.js

```diff
@@ -4,7 +4,7 @@
 import {connect} from 'react-redux';
 import {bindActionCreators} from 'redux';
 import {getBool} from 'mattermost-redux/selectors/entities/preferences';
-import {resetHistoryIndex} from 'mattermost-redux/actions/posts';
+import {resetCreatePost, resetHistoryIndex} from 'mattermost-redux/actions/posts';
```

Comment:
I don't see this action in the mattermost-redux project, is this something that you need to add ?

Reply:
:) I'm still preparing for the redux PR, just finishing the test

Reply:
ok sounds good, but can I ask you to submit the PR to redux first and then add the link to the PR in the description? I went on looking for that function in the mattermost-redux repo for a while and found nothing

Reply:
Sorry to keep you waiting. I had trouble writing test on mattermost-redux, only to find out that I need to add ``await TestHelper.wait(300);`` :(